storybookjs/storybook
# documentation: Clarify description of args - Add example of args controlling more than props

Issue:

There were a number of questions in the vue discord channel inquiring how to control non-props aspects of components in stories (i.e. slots). I tried to update the args text to be more encompassing and make it clear that all dynamic changes to a template go through the single `args` object.

## What I did

- Updated args intro
- Added section/example for adding slots via an arg

## How to test

- Is this testable with Jest or Chromatic screenshots?
No
- Does this need a new example in the kitchen sink apps?
No
- Does this need an update to the documentation?
It is one 😄 

If your answer is yes to any of these, please make sure to include it in your PR.
